### ISSUE TITLE
Intermittent timeout in /websockets/opening-handshake/003.html

### ISSUE THREAD

**Issue by jdm** [servo/servo#9777]
```
  ▶ TIMEOUT [expected OK] /websockets/opening-handshake/003.html

  ▶ Unexpected subtest result in /websockets/opening-handshake/003.html:
  │ TIMEOUT [expected PASS] WebSockets: origin
  └   → Test timed out
```

**Comment by perlun** [comment]
Any idea of what could be causing it? (In other words: Anywhere I can start digging? :smile:)

**Comment by jdm** [comment]
Yes-ish: https://github.com/servo/servo/issues/9803#issuecomment-196424406

**Comment by jdm** [comment]
Should have been fixed by #10240.
